layers/+tools/bm/packages.el

```diff
@@ -0,0 +1,57 @@
+;;; packages.el --- bm Layer packages File
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Eugene "JAremko" Yaremenko <w3techplayground@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq bm-packages '(bm))
+
+(defun bm/init-bm ()
+  "initializes bm-emacs and adds a key binding to `SPC f z'"
+  (use-package bm
+    :defer t
+    :commands (bm-buffer-restore)
+    :init (progn
+            ;; Allow cross-buffer 'next'
+            (setq bm-cycle-all-buffers t)
+            ;; save bookmarks
+            (setq-default bm-buffer-persistence t)
+            ;; where to store persistant files
+            (setq bm-repository-file (format "%sbm-repository"
+                                             spacemacs-cache-directory))
+            (spacemacs|define-transient-state bm
+              :title "BM Transient State"
+              :doc "
+ Go to bookmark^^              Toggle^^          Other^^
+ ──────────────^^───────────── ──────^^───────── ─────^^───
+ [_n_/_N_] next/previous [_t_] bookmark at point [_q_] quit"
```

Comment:
@duianto  Ok ok mr. pixelperfect we all  gonna fix all of those critical bugs. Ok.. :roll_eyes: 

:smile: